portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,10 +28,12 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
+  private int quantity; // Keeps track of number of desired comments to be shown 
```

Comment:
Can you move all comments to a new line right above the variable/logic that they address? (similar to how you have it at line 84 below).